src/Row.tsx

```diff
@@ -71,23 +76,25 @@ function Row<R, SR = unknown>({
         }
 
         return (
-          <CellRenderer
-            key={column.key}
-            rowIdx={rowIdx}
-            column={column}
-            row={row}
-            isCopied={copiedCellIdx === column.idx}
-            isDraggedOver={draggedOverCellIdx === column.idx}
-            isCellSelected={isCellSelected}
-            isRowSelected={isRowSelected}
-            dragHandleProps={isCellSelected ? (selectedCellProps as SelectedCellProps).dragHandleProps : undefined}
-            onFocus={isCellSelected ? (selectedCellProps as SelectedCellProps).onFocus : undefined}
-            onKeyDown={isCellSelected ? selectedCellProps!.onKeyDown : undefined}
-            onRowClick={onRowClick}
-            onRowChange={onRowChange}
-            selectCell={selectCell}
-            selectRow={selectRow}
-          />
+          <RowSelectionChangeContext.Provider value={onRowSelectionChange}>
+            <RowSelectionContext.Provider value={isRowSelected}>
```

Comment:
Creating 2 Provider instance per cell seems overkill, can't we just have 2 per row by rendering them straight under the row's `div`?

Reply:
We should. Totally missed it